doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/migration_guide_2.4.99-cs3_to_2.7_application.rst

```diff
@@ -74,6 +74,34 @@ General
 
      The previously used SOC1-based board files have been removed.
 
+CoreMark
+--------
+
+Several changes have been made to migrate the :ref:`coremark_sample` sample from the nRF54 customer sampling release v2.4.99-cs3 to |NCS| v2.7.0:
+
+* The sample has been moved from the :file:`<ncs_path>/zephyr/samples/benchmarks/coremark` folder in the Zephyr repository to the :file:`<ncs_path>/nrf/samples/benchmarks/coremark` folder in the main |NCS| repository.
+* The build system has been aligned to the hardware model v2.
+  The ``SB_CONFIG_IMAGE_2_BOARD`` and ``SB_CONFIG_IMAGE_3_BOARD`` Kconfig options have been replaced with the ``SB_CONFIG_APP_CPUNET_RUN`` and ``SB_CONFIG_APP_CPUPPR_RUN`` Kconfig options.
+  The new Kconfig options more clearly indicate on which cores the sample will be run.
+  See the :file:`sysbuild.cmake` file to find out how these options are used to determine the board target.
+* Because the |NCS| v2.7.0 does not support ARM Coresight System Trace Macrocell (STM) logging for the nRF54 device, STM logging has been removed from the sample.
+  The sample now uses usual UART logging that allows for sending logs from only one core for each UART instance.
+  The nRF54 device has only two UART instances, so the sample now can be run on two cores at most.
+  The sample is always run on application core and depending on configuration can be run on either the radio core or the PPR core.
```

Comment:
```suggestion
  The sample is always run on the application core, and depending on configuration, it can be run on either the radio core or the PPR core.
```